x-pack/plugins/observability/public/components/app/burn_rate_rule_editor/long_window_duration.tsx

```diff
@@ -10,110 +10,80 @@ import {
+  shortWindowDuration,
+  initialDuration,
+  onChange,
+  errors,
+}: Props) {
   const [durationValue, setDurationValue] = useState<number>(initialDuration?.value ?? 1);
-  const [durationUnit, setDurationUnit] = useState<DurationUnit>(
-    initialDuration?.unit ?? durationUnitOptions[0].value
-  );
-  const [error, setError] = useState<string | undefined>(undefined);
+  const hasError = errors !== undefined && errors.length > 0;
 
   const onDurationValueChange = (e: ChangeEvent<HTMLInputElement>) => {
-    const value = parseInt(e.target.value, 10);
-    setDurationValue(!isNaN(value) ? value : 1);
-    onChange({ value, unit: durationUnit });
-  };
-
-  const onDurationUnitChange = (e: ChangeEvent<HTMLSelectElement>) => {
-    const unit = e.target.value === 'm' ? 'm' : 'h';
-    setDurationUnit(unit);
-    onChange({ value: durationValue, unit });
-  };
-
-  useEffect(() => {
-    if (isValidDuration(durationValue, durationUnit)) {
-      setError(undefined);
-    } else {
-      setError(errorText);
-    }
-  }, [durationValue, durationUnit]);
-
-  const isValidDuration = (value: number, unit: DurationUnit): boolean => {
-    return (
-      (unit === 'm' && value >= MIN_DURATION_IN_MINUTES && value <= MAX_DURATION_IN_MINUTES) ||
-      (unit === 'h' && value <= MAX_DURATION_IN_HOURS)
-    );
+    const value = Number(e.target.value);
+    setDurationValue(value);
+    onChange({ value, unit: 'h' });
   };
 
-  const selectId = useGeneratedHtmlId({ prefix: 'durationUnitSelect' });
   return (
-    <EuiFormRow label={rowLabel} fullWidth isInvalid={!!error} error={error}>
+    <EuiFormRow
+      label={getRowLabel(shortWindowDuration)}
+      fullWidth
+      isInvalid={hasError}
+      error={hasError ? errors[0] : undefined}
+    >
       <EuiFlexGroup direction="row">
-        <EuiFlexItem grow={false} style={{ width: 100 }}>
+        <EuiFlexItem>
           <EuiFieldNumber
-            isInvalid={!!error}
+            isInvalid={hasError}
             min={1}
+            max={24}
+            step={1}
             value={String(durationValue)}
             onChange={onDurationValueChange}
             aria-label={valueLabel}
             data-test-subj="durationValueInput"
           />
         </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiSelect
-            id={selectId}
-            isInvalid={!!error}
-            options={durationUnitOptions}
-            value={durationUnit}
-            onChange={onDurationUnitChange}
-            aria-label={unitLabel}
-            data-test-subj="durationUnitSelect"
-          />
-        </EuiFlexItem>
       </EuiFlexGroup>
     </EuiFormRow>
   );
 }
 
-const rowLabel = i18n.translate('xpack.observability.slo.rules.longWindow.rowLabel', {
-  defaultMessage: 'Long window',
-});
+const getRowLabel = (shortWindowDuration: Duration) => (
+  <>
+    {i18n.translate('xpack.observability.slo.rules.longWindow.rowLabel', {
+      defaultMessage: 'Lookback period (hours)',
+    })}{' '}
```

Comment:
Is the white space `{'  '}` an prettier artefact or required?

Reply:
It's a prettier thing that requires it :/